src/interactableComponents/drawer/index.js

```diff
@@ -441,11 +447,12 @@ export default class Drawer extends BaseComponent {
     );
   }
   render() {
-    const {style, onPress, rightItems} = this.props;
+    const {style, onPress, rightItems} = this.getThemeProps();
     const Container = onPress ? TouchableOpacity : View;
+    const backgroundColor = rightItems ? rightItems[0].background || ITEM_BG : ITEM_BG;
```

Comment:
with `_.get` you can do this line pretty easily
`_.get(rightItems, '[0].background', ITEM_BG)`, the last param is the default in case what you're looking for is undefined